its/pom.xml

```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>org.sonarsource.scanner.ant</groupId>
     <artifactId>ant</artifactId>
-    <version>2.7.1-SNAPSHOT</version>
+    <version>2.7.2-SNAPSHOT</version>
```

Comment:
it seems to me that you didn't update the good file. this is only the `its`. Did you use the command?
```
mvn versions:set -DgenerateBackupPoms=false -DnewVersion=2.7.2-SNAPSHOT
```

Reply:
Oops, I wasn't aware of that command. Fixed.

Reply:
FYI that's the one that we recommmend in our release-processes (example: [generic](https://xtranet-sonarsource.atlassian.net/wiki/spaces/DEV/pages/776802/Release+Procedures), [sonar-java specific](https://xtranet-sonarsource.atlassian.net/wiki/spaces/LANG/pages/2088861731/Release+sonar-java))